docs/deploying.md

```diff
@@ -17,7 +17,6 @@ Create a folder for your registry data:
 Start your registry:
 
      $ docker run -d -p 5000:5000 \
-     		-v `pwd`/registry-data:/tmp/registry-dev \
```

Comment:
Removing this since the paragraph below covers the topic of data persistence.
Also, once we merge the Dockerfile default volume mount, this would be moot.